README.md

```diff
@@ -269,7 +269,7 @@ Empty the Calender.
 
 ### Event
 
-#### uid([_String_|_Number_ uid]) or id([_String_|_Number_ id])
+#### uid([_String_ uid]) or id([_String_ id])
```

Comment:
Oops, I didn't know the `uid` change slipped into this PR.

`uid` has multiple type definitions that describe it only returning and accepting strings, but now that I look at the tests and code, I'm guessing the typings are the one needing updating. I will revert this line and open an issue for discussion.

Reply:
Opened #133